book/docs/setup/editors/vscode.md

```diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
    ```
    _Note: if you're using `nix-shell`, make sure to run it first._
 
-If everything is fine you should get auto-completion and other features working 🎉
\ No newline at end of file
+If everything is fine you should get auto-completion and other features working 🎉
+
+---
+</br>
```

Comment:
</br> redundant

Reply:
Bad habits die hard. I have removed the `</br>` line break. Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.